package.json

```
@@ -33,48 +36,49 @@
   "homepage": "https://github.com/uptick/react-keyed-file-browser#readme",
   "dependencies": {
```

These loaders have a minimum required node version, we should probably be specifying something to that effect
```suggestion
  "engines": {
    "node": ">=10.13.0",
  },
  "dependencies": {
```